src/OrbitGl/App.cpp

```diff
@@ -2952,17 +2954,19 @@ void OrbitApp::AddFrameTrack(const FunctionInfo& function) {
 }
 
 void OrbitApp::AddFrameTrack(uint64_t instrumented_function_id) {
-  ORBIT_CHECK(instrumented_function_id != 0);
```

Comment:
Why is this check being removed? I think it's beneficial even if `ORBIT_CHECK(scope_id.has_value());` is added.

Reply:
I removed it because, as you have pointed out, invalid function id leads to an empty optional returned from `ProvideScopeId`. But let's get it back. Naturally, I replace the `0` with an appropriate constant (which is yet to be eradicated!).